crs_support/urdf/crs.urdf.xacro

```diff
@@ -433,45 +419,19 @@
           </geometry>
           <origin rpy="0 0 0" xyz="0 0 0" />
       </collision>
-      <collision>
-          <geometry>
-              <cylinder length="0.03" radius="0.115" />
-          </geometry>
-          <origin rpy="0 0 0" xyz="0 0 -0.105" />
-      </collision>
-<!--      <collision>
-          <geometry>
-              <cylinder length="0.025" radius="0.135" />
-          </geometry>
-          <origin rpy="0 0 0" xyz="0 0 -0.035" />
-      </collision>-->
     </link>
-    <joint name="${prefix}sander_joint" type = "fixed">
+
+    <joint name="${prefix}sander_intermediate_to_sander_center_link" type="fixed">
+      <parent link="${prefix}sander_intermediate_link" />
+      <child link="${prefix}sander_center_link" />
+      <origin  xyz="0.0 0.0 0.0125" rpy="0.0 0.0 0.0"/>
+    </joint>
+
+    <joint name="${prefix}tool0_to_sander_intermediate_link" type="fixed">
+        <parent link="${prefix}eoat_link" />
+        <child link="${prefix}sander_intermediate_link" />
```

Comment:
Is this the configuration for the mockup end effector?

Reply:
I added this intermediate link to account for the sander pad which I am uncertain of it's size and this makes it easier to modify/account for that and I can easily add sander pad collision geometry. This is for the primary Spirit end effector, not the test sander bracket we are using at SwRI